#these are in mm and are the point position of the person
float32 target_x
float32 target_y
float32 target_z
